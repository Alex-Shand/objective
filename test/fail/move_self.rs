#[objective::objective]
trait Test {
    fn test(self);
}

fn main() {}
