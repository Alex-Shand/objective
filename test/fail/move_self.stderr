error: All trait methods must accept &self or &mut self
 --> test/fail/move_self.rs:3:13
  |
3 |     fn test(self);
  |             ^^^^
